codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpProtocolUnitTestGenerator.java

```diff
@@ -58,19 +62,22 @@
     protected final Symbol outputSymbol;
     protected final String protocolName;
     protected final Set<ConfigValue> clientConfigValues = new TreeSet<>();
+    protected final Set<SkipTest> skipTests = new TreeSet<>();
 
     /**
      * Initializes the abstract protocol tests generator.
      *
      * @param builder the builder initializing the generator.
      */
     protected HttpProtocolUnitTestGenerator(Builder<T> builder) {
-        this.model = builder.model;
-        this.symbolProvider = builder.symbolProvider;
-        this.protocolName = builder.protocolName;
-        this.operation = builder.operation;
-        this.testCases = builder.testCases;
+        this.model = SmithyBuilder.requiredState("model", builder.model);
```

Comment:
I believe the first argument string is meant to match the property setter name. So instead of “symbol provider” it should be “symbolProvider”

Reply:
I seen, wasn't sure about that can fix that. I assumed it was for a message, but makes sense to match the field name directly.